packages/fx-core/src/component/driver/aad/create.ts

```diff
@@ -237,14 +230,4 @@ export class CreateAadAppDriver implements StepDriver {
     state.authorityHost = Constants.oauthAuthorityPrefix;
     state.authority = `${Constants.oauthAuthorityPrefix}/${tenantId}`;
   }
-
-  private getProgressBarSetting(): ProgressBarSetting {
-    return {
-      title: getLocalizedString("driver.aadApp.progressBar.createAadAppTitle"),
-      stepMessages: [
-        getLocalizedString("driver.aadApp.progressBar.createAadAppStepMessage"), // step 1
```

Comment:
can we delete the unused messages from package.nls.json?

Reply:
Fixed. Thanks.